rpcs3/Emu/RSX/Program/GLSLSnippets/VideoOutCalibrationPass.glsl

```diff
@@ -0,0 +1,157 @@
+R"(
+#version 420
+
+#ifdef VULKAN
+layout(set=0, binding=1) uniform sampler2D fs0;
+layout(set=0, binding=2) uniform sampler2D fs1;
+#else
+layout(binding=31) uniform sampler2D fs0;
+layout(binding=30) uniform sampler2D fs1;
+#endif
+
+layout(location=0) in vec2 tc0;
+layout(location=0) out vec4 ocol;
+
+#define STEREO_MODE_DISABLED 0
+#define STEREO_MODE_SIDE_BY_SIDE 1
+#define STEREO_MODE_OVER_UNDER 2
+#define STEREO_MODE_INTERLACED 3
+#define STEREO_MODE_ANAGLYPH_RED_GREEN 4
+#define STEREO_MODE_ANAGLYPH_RED_BLUE 5
+#define STEREO_MODE_ANAGLYPH_RED_CYAN 6
+#define STEREO_MODE_ANAGLYPH_MAGENTA_CYAN 7
+#define STEREO_MODE_ANAGLYPH_TRIOSCOPIC 8
+#define STEREO_MODE_ANAGLYPH_AMBER_BLUE 9
+
+vec2 left_single_matrix  = vec2(1.f, 0.4898f);
+vec2 right_single_matrix = vec2(0.f, 0.510204f);
+vec2 sbs_single_matrix   = vec2(2.0, 0.4898f);
+vec2 sbs_multi_matrix    = vec2(2.0, 1.0);
+vec2 ou_single_matrix    = vec2(1.0, 0.9796f);
+vec2 ou_multi_matrix     = vec2(1.0, 2.0);
+
+#ifdef VULKAN
+layout(push_constant) uniform static_data
+{
+	float gamma;
+	int limit_range;
+	int stereo_display_mode;
+	int stereo_image_count;
+	int height;
+};
+#else
+uniform float gamma;
+uniform int limit_range;
+uniform int stereo_display_mode;
+uniform int stereo_image_count;
+uniform int height;
+#endif
+
+vec4 anaglyph(vec4 left, vec4 right)
```

Comment:
```suggestion
vec4 anaglyph(const in vec4 left, const in vec4 right)
```
Always decorate with const in. It generates more readable SPIR-V and consequently any decompiled glsl from tools like renderdoc.